x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/MlDailyMaintenanceService.java

```diff
@@ -151,6 +151,10 @@ private void triggerTasks() {
                 LOGGER.warn("skipping scheduled [ML] maintenance tasks because upgrade mode is enabled");
                 return;
             }
+            if (MlMetadata.getMlMetadata(clusterService.state()).isResetMode()) {
+                LOGGER.warn("skipping scheduled [ML] maintenance tasks because reset mode is enabled");
```

Comment:
```suggestion
                LOGGER.warn("skipping scheduled [ML] maintenance tasks because machine learning feature reset is in progress");
```

(Because "reset mode" won't be a documented thing, but the "feature reset API" will be.)